app/models/data_store_record.rb

```diff
@@ -0,0 +1,9 @@
+require "flex_commerce_api/api_base"
+
+module FlexCommerce
+  class DataStoreRecord < FlexCommerceApi::ApiBase
+    def self.table_name
+      'generic_data_store/record'
+    end
```

Comment:
We may be better changing the type on the API-side though. `'type': 'record'` makes sense, but `'type': 'type'` (for `DataStoreType`) may be confusing.